src/main/java/com/backbase/ct/bbfuel/setup/AccessControlSetup.java

```diff
@@ -77,12 +79,16 @@
     private final JobProfileReader jobProfileReader;
     private final ProductGroupSeedReader productGroupSeedReader;
     private final LegalEntityService legalEntityService;
+    private final ServiceAgreementsIntegrationRestClient serviceAgreementsIntegrationRestClient;
     @Getter
     private List<LegalEntityWithUsers> legalEntitiesWithUsers;
     @Setter
     private List<JobProfile> jobProfileTemplates;
     private List<ProductGroupSeed> productGroupSeedTemplates;
 
+    private Predicate<JobProfile> isTemplate =  jobProfile -> jobProfile.getType().equals("TEMPLATE");
+    private Predicate<String> isRootServiceAgreement = serviceAgreementName -> serviceAgreementName.equals("Bank");
```

Comment:
These predicates are good candidates to be `static final` as well and be renamed to: `JOB_PROFILE_IS_TEMPLATE` and `SERVICE_AGREEMENT_NAME_IS_BANK`.

Reply:
Done.